src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/ProxyStreamFactory.java

```diff
@@ -605,6 +608,19 @@ private void sendHttpResponse(
                 final ListFW<HttpHeaderFW> responseHeaders,
                 final ListFW<HttpHeaderFW> requestHeaders)
         {
+            if (canInjectPushPromise(requestHeaders)
```

Comment:
I don't understand why there is a call here to canInjectPushPromise.

Furthermore, This is not correct.  The presence of max-stale does not mean the response is stale

```
 The "max-stale" request directive indicates that the client is
   willing to accept a response that has exceeded its freshness
   lifetime.
```

It does not mean that the response has to be stale.